# This is an easy patch to add DESTDIR support to the Makefiles in splix
# Written by Robby Workman - http://rlworkman.net
# Permission is granted to use the patch as you see fit

diff -Nur splix-1.1.1.orig/ppd/Makefile splix-1.1.1/ppd/Makefile
--- splix-1.1.0.orig/ppd/Makefile
+++ splix-1.1.0/ppd/Makefile
@@ -62,25 +62,25 @@
 
 .PHONY: install
 install:
-	install -d -m 755 ${CUPSPPD}/samsung
+	install -d -m 755 $(DESTDIR)/${CUPSPPD}/samsung
 	for filename in ${SAMSUNG}; do \
-		install -m 644 $$filename.${DRIVERSEXT} ${CUPSPPD}/samsung;\
+		install -m 644 $$filename.${DRIVERSEXT} $(DESTDIR)/${CUPSPPD}/samsung;\
 		for lang in ${LANGUAGES}; do \
-			install -m 644 $$filename$$lang.${DRIVERSEXT} ${CUPSPPD}/samsung;\
+			install -m 644 $$filename$$lang.${DRIVERSEXT} $(DESTDIR)/${CUPSPPD}/samsung;\
 		done; \
 	done; \
-	install -d -m 755 ${CUPSPPD}/xerox
+	install -d -m 755 $(DESTDIR)/${CUPSPPD}/xerox
 	for filename in ${XEROX}; do \
-		install -m 644 $$filename.${DRIVERSEXT} ${CUPSPPD}/xerox;\
+		install -m 644 $$filename.${DRIVERSEXT} $(DESTDIR)/${CUPSPPD}/xerox;\
 		for lang in ${LANGUAGES}; do \
-			install -m 644 $$filename$$lang.${DRIVERSEXT} ${CUPSPPD}/xerox;\
+			install -m 644 $$filename$$lang.${DRIVERSEXT} $(DESTDIR)/${CUPSPPD}/xerox;\
 		done; \
 	done; \
-	install -d -m 755 ${CUPSPPD}/dell
+	install -d -m 755 $(DESTDIR)/${CUPSPPD}/dell
 	for filename in ${DELL}; do \
-		install -m 644 $$filename.${DRIVERSEXT} ${CUPSPPD}/dell;\
+		install -m 644 $$filename.${DRIVERSEXT} $(DESTDIR)/${CUPSPPD}/dell;\
 		for lang in ${LANGUAGES}; do \
-			install -m 644 $$filename$$lang.${DRIVERSEXT} ${CUPSPPD}/dell;\
+			install -m 644 $$filename$$lang.${DRIVERSEXT} $(DESTDIR)/${CUPSPPD}/dell;\
 		done; \
 	done;
 
diff -Nur splix-1.1.1.orig/src/Makefile splix-1.1.1/src/Makefile
--- splix-1.1.1.orig/src/Makefile
+++ splix-1.1.1/src/Makefile
@@ -30,7 +30,7 @@
 	$(CXX) $(CXXFLAGS) -c $<
 
 install: rastertospl2
-	install -m 755 -s rastertospl2 ${CUPSFILTER}
+	install -m 755 -s rastertospl2 $(DESTDIR)/${CUPSFILTER}
 
 .PHONY: clean distclean
 clean:
